book/website/collaboration/research-infrastructure-roles/community-manager.md

```diff
@@ -0,0 +1,89 @@
+(cl-infrastructure-communitymanagers)=
+# Community Managers: Overview
+
+Community Managers are a growing role within scientific communities. 
+They organise, support, maintain, grow, develop, evaluate and collaborate with people involved in research and scientific communities.
+
+(cl-infrastructure-communitymanagers-tasks)=
+## What do Community Managers do? 
+
+Community Managers main objective is to organise groups of scientists, researchers and/or patients and the public around shared research topics and objectives. 
+They are often employed by professional societies, universities, research institutions, larger programmes, and non-profit organisations. 
+
+Each role is varied but the main activities are typically focused around compassion, stewardship, and collaboration with the community: 
+* Catalyse connections with all stakeholders
+* Foster communities (integrating communities together)
+* Linking the right people up together (based on expertise and interests)
+* Active community member
+* Communicating technical topics to non-technical colleagues and vice versa
+* Encouraging high standards of reproducible, ethical, inclusive and collaborative data science
+
+The day-to-day tasks of a community manager could include: 
+* Organising and hosting community calls
+* Onboarding new members
+* Attending community leadership meetings
+* Writing community reports or newsletters
```

Comment:
could add 'creating informational materials to help others learn about the community and their projects'

Reply:
done!